src/components/Genre/Genre.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { fetchSingleGenre } from '../../utils/API/ApiFetch';
-import Card from '../../containers/Card/Card';
+import Card from '../Card/Card';
```

Comment:
Nice job reviewing the need for Card to MSTP and MDTP, noticing it did not need to be mapping and changing it to a component.